docs/spark.rst

```diff
@@ -83,7 +86,7 @@ leveraging Horovod’s ability to scale across multiple workers, without any spe
         .setOutputCols(['predict'])
     predict_df = keras_model.transform(test_df)
 
-The Estimator hides the complexity of gluing Spark DataFrames to a deep learning training script, reading data into a
+The `KerasEstimator <https://github.com/horovod/horovod/blob/master/horovod/spark/keras/estimator.py#L98>`__ hides the complexity of gluing Spark DataFrames to a deep learning training script, reading data into a
```

Comment:
```suggestion
The Estimator hides the complexity of gluing Spark DataFrames to a deep learning training script, reading data into a
```

I'd prefer to leave this line as it is. (a) There are more subclasses of `Estimator`, e.g. for PyTorch rather than Keras. (b) Pointing to a specific source line on master is fragile, and people will not remember to update this link.